composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "jonnyw/php-phantomjs",
-    "description": "A PHP wrapper for loading pages through PhantomJS",
+    "name": "cocostru/php-phantomjs",
```

Comment:
You should remove this and similar modifications in order to consider merge of this PR.